lib/jazzy/doc_builder.rb

```diff
@@ -113,6 +113,10 @@ def self.build_docs_for_sourcekitten_output(sourcekitten_output, options)
         options.skip_undocumented,
       )
 
+      if options.lint_report
```

Comment:
The intent is to have the lint report be a separate phase from the doc generation (see #393).